.changeset/all-loops-strive.md

```diff
@@ -0,0 +1,6 @@
+---
+"@biomejs/biome": minor
+---
+
+Enhanced no_focused_test rule to also recognize usage in loops. Solves #5493
+Implement new helper methods in `biome_js_syntax/src/expr_ext.rs` to search for focused tests with or without loops.
```

Comment:
```suggestion
```

We usually document only the changes for our end users. We don't document changes to our Rust crates (for now).